PharoObjectModel/PharoObjectModel.pier

```diff
@@ -292,56 +296,66 @@ Dog class>>>count
     ^ count
 ]]]
 
```

Comment:
The definition is right below it - the problem is, Github only shows 3 lines of context by default :)
There's a little 'unfold' icon on the left, (below the line number '264') that you can use to see the rest of the code.